ksql-streams/src/test/java/io/confluent/ksql/execution/streams/AggregateParamsFactoryTest.java

```diff
@@ -132,10 +140,21 @@ public void init() {
   @SuppressWarnings("unchecked")
   @Test
   public void shouldCreateAggregatorWithCorrectParams() {
-    verify(udafFactory).create(
-        ImmutableList.of(0, 2),
```

Comment:
The `NON_AGG_COLUMNS` example in this unit test file confuses me since it seems to suggest non-aggregate columns may be intermingled with aggregate columns in the input, and violates the assumption that enables the refactor in this PR. I assume it's just the unit test that needs to be updated?

Reply:
Yeah, just the test out of whack. It's ok now though, right?

Reply:
Did anything change here? I don't see an update: https://github.com/confluentinc/ksql/blob/master/ksql-streams/src/test/java/io/confluent/ksql/execution/streams/AggregateParamsFactoryTest.java#L54-L55